providers/org/datacite/normalizer.py

```diff
@@ -28,243 +74,402 @@ def force_text(data):
     raise Exception('{} is not a string or a dictionary.'.format(data))
 
 
-class Link(Parser):
-    url = tools.DOI(tools.RunPython(force_text, ctx))
-    type = tools.Static('doi')
+def get_agent_type(agent, default='Person'):
+    """
+    Returns agent type based on contributor type.
+    """
+    is_not_person = try_contributor_type(agent, NOT_PEOPLE_TYPES)
+    is_person = try_contributor_type(agent, PEOPLE_TYPES)
+
+    if is_person:
+        return 'Person'
+    elif is_not_person:
+        return 'Organization'
```

Comment:
Could any of the `NOT_PEOPLE_TYPES` be an institution? Not sure if `HostingInstitution` means it the same way we do.